providers/fab/src/airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -408,7 +408,7 @@ def security_manager(self) -> FabAirflowSecurityManagerOverride:
 
     def get_url_login(self, **kwargs) -> str:
         """Return the login page url."""
-        return f"{self.apiserver_endpoint}/auth/login"
+        return f"{self.apiserver_endpoint}/auth/login/"
```

Comment:
This is outside the scope of this PR but introduced an extra redirection. This will directly land on the trailing slash url that is the correct one.